--- tools/build/v2/tools/darwin.jam.old	2011-05-07 13:10:50.000000000 -0700
+++ tools/build/v2/tools/darwin.jam	2011-05-07 13:10:55.000000000 -0700
@@ -482,7 +482,7 @@
 flags darwin.compile OPTIONS <link>shared : -dynamic ;
 
 # Misc options.
-flags darwin.compile OPTIONS : -no-cpp-precomp -gdwarf-2 ;
+flags darwin.compile OPTIONS : -gdwarf-2 ;
 
 # Add the framework names to use.
 flags darwin.link FRAMEWORK <framework> ;
